template/crusher_finalize.jinja

```diff
@@ -0,0 +1,5 @@
+# cupy does not support the cuda array interface on crusher
+pip uninstall -y cupy
```

Comment:
Yep, CuPy is NVIDIA specific for now.

Reply:
CuPy builds for rocm successfully, but when the HOOMD unit tests use the `__cuda_array_interface__`, it gives an error:
https://docs.cupy.dev/en/stable/install.html#building-cupy-for-rocm-from-source
https://docs.cupy.dev/en/stable/install.html#limitations

Reply:
Oh that is news for me, they used to be CUDA only, but yeah not supporting `__cuda_array_interface__` is a non-starter.